labelbox/schema/bulk_import_request.py

```diff
@@ -0,0 +1,76 @@
+import json
+from pathlib import Path
+from typing import Iterable
+
+import ndjson
+
+from labelbox import Client
+from labelbox.exceptions import LabelboxError
+from labelbox.orm import query
+from labelbox.orm.db_object import DbObject
+from labelbox.orm.model import Field
+from labelbox.orm.model import Relationship
+from labelbox.schema.enums import BulkImportRequestState
+from labelbox.schema.enums import UploadedFileType
+
+
+class BulkImportRequest(DbObject):
```

Comment:
i think worth adding tests for this

Reply:
Definitely:) Will work on this after API pr is merged